src/main/java/org/primefaces/event/data/SortEvent.java

```diff
@@ -23,31 +23,44 @@
  */
 package org.primefaces.event.data;
 
-import javax.faces.component.UIComponent;
-import javax.faces.component.behavior.Behavior;
 import org.primefaces.component.api.UIColumn;
 import org.primefaces.event.AbstractAjaxBehaviorEvent;
 import org.primefaces.model.SortOrder;
 
+import javax.faces.component.UIComponent;
+import javax.faces.component.behavior.Behavior;
+
```

Comment:
Please put these imports back they way they were it looks like you just formatted the file.  javax. should always come before org. primefaces